x-pack/plugins/secops/public/components/empty_string/index.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import * as i18n from './translations';
+
+export const EmptyString = styled.span`
```

Comment:
Doesn't appear as if you have any direct usages currently -- is this exported with intent for it to be used on its own in certain circumstances, or should all usage be directed through the `getEmptyStringTag` and `getOrEmptyStringTag` functions?

Reply:
It's a habit. I should not export it if I don't need to.